numpy/random/mtrand/mtrand.pyx

```diff
@@ -1551,18 +1551,17 @@ cdef class RandomState:
         cdef double flow, fhigh, fscale
         cdef object temp
 
-        flow = PyFloat_AsDouble(low)
-        fhigh = PyFloat_AsDouble(high)
+        if np.array(low).shape == np.array(high).shape == ():
```

Comment:
Is this double comparison valid Cython? Or does it end comparing `True` (or `False`) to `()`?